kerl

```diff
@@ -1210,73 +1210,9 @@ emit_activate() {
   cat <<\
 =======
 #!/bin/sh
-# credits to virtualenv
-kerl_deactivate() {
-    set -o allexport
-    if [ -n "\${_KERL_ERL_AFLAGS_SET+x}" ]; then
-        if [ -n "\$_KERL_ERL_AFLAGS_SET" ]; then
-            ERL_AFLAGS="\$_KERL_SAVED_ERL_AFLAGS"
-        else
-            unset ERL_AFLAGS
-        fi
-        unset _KERL_SAVED_ERL_AFLAGS
-        unset _KERL_ERL_AFLAGS_SET
-    fi
-    if [ -n "\$_KERL_PATH_REMOVABLE" ]; then
-        # shellcheck disable=SC2001
-        PATH="\$(echo "\$PATH" | sed -e "s#\$_KERL_PATH_REMOVABLE:##")"
-        unset _KERL_PATH_REMOVABLE
-    fi
-    if [ -n "\$_KERL_MANPATH_REMOVABLE" ]; then
-        # shellcheck disable=SC2001
-        MANPATH="\$(echo "\$MANPATH" | sed -e "s#\$_KERL_MANPATH_REMOVABLE:##")"
-        if [ -z "\$MANPATH" -a -z "\$_KERL_MANPATH_SET" ]; then
-            unset MANPATH
-        fi
-        unset _KERL_MANPATH_REMOVABLE
-    fi
-    unset _KERL_MANPATH_SET
-    if [ -n "\${_KERL_REBAR_PLT_DIR_SET+x}" ]; then
-        if [ -n "\$_KERL_REBAR_PLT_DIR_SET" ]; then
-            REBAR_PLT_DIR="\$_KERL_SAVED_REBAR_PLT_DIR"
-        else
-            unset REBAR_PLT_DIR
-        fi
-        unset _KERL_SAVED_REBAR_PLT_DIR
-        unset _KERL_REBAR_PLT_DIR_SET
-    fi
-    if [ -n "\${_KERL_REBAR_CACHE_DIR_SET+x}" ]; then
-        if [ -n "\$_KERL_REBAR_CACHE_DIR_SET" ]; then
-            REBAR_CACHE_DIR="\$_KERL_SAVED_REBAR_CACHE_DIR"
-        else
-            unset REBAR_CACHE_DIR
-        fi
-        unset _KERL_SAVED_REBAR_CACHE_DIR
-        unset _KERL_REBAR_CACHE_DIR_SET
-    fi
-    if [ -n "\$_KERL_ACTIVE_DIR" ]; then
-        unset _KERL_ACTIVE_DIR
-    fi
-    if [ -n "\$_KERL_SAVED_PS1" ]; then
-        PS1="\$_KERL_SAVED_PS1"
-        unset _KERL_SAVED_PS1
-    fi
-    if [ -n "\$_KERL_ERL_CALL_REMOVABLE" ]; then
-        # shellcheck disable=SC2001
-        PATH="\$(echo "\$PATH" | sed -e "s#\$_KERL_ERL_CALL_REMOVABLE:##")"
-    fi
-    unset _KERL_ERL_CALL_REMOVABLE
-    if [ -n "\$BASH" ] || [ -n "\$ZSH_VERSION" ]; then
-        hash -r
-    fi
-    if [ ! "\$1" = "nondestructive" ]; then
-        unset -f kerl_deactivate
-    fi
-    unset KERL_ENABLE_PROMPT
-    unset KERL_PROMPT_FORMAT
-    set +o allexport
-}
-kerl_deactivate nondestructive
+if type kerl_deactivate 2>/dev/null | \grep -qw function; then
+    kerl_deactivate
+fi
 
 _KERL=$(command -v kerl)
 
```

Comment:
We should remove white space -only lines from here until `unset _KERL_VERSION _KERL`. This way we "identify" stuff as blocks, hopefully increasing readability.

Reply:
i.e.

```bash
_KERL=$(command -v kerl)

if [ -n "\$_KERL" ]; then
  _KERL_VERSION=\$(\$_KERL version)
fi

if [ -n "\$_KERL_VERSION" ] && [ "\$_KERL_VERSION" != "$KERL_VERSION" ]; then
  echo "WARNING: this Erlang/OTP installation appears to be stale. Please consider reinstalling."
  echo "         It was created with kerl $KERL_VERSION, and the current version is \$_KERL_VERSION."
fi

unset _KERL_VERSION _KERL
```

becomes

```bash
_KERL=$(command -v kerl)
if [ -n "\$_KERL" ]; then
  _KERL_VERSION=\$(\$_KERL version)
fi
if [ -n "\$_KERL_VERSION" ] && [ "\$_KERL_VERSION" != "$KERL_VERSION" ]; then
  echo "WARNING: this Erlang/OTP installation appears to be stale. Please consider reinstalling."
  echo "         It was created with kerl $KERL_VERSION, and the current version is \$_KERL_VERSION."
fi
unset _KERL_VERSION _KERL
```

but I can't suggest this via the GH interface 😢